### PR TITLE
[MOS-28] Change alarm sounds and ringtones preview behavior

### PR DESCRIPTION
**Pull request by Lefucjusz**
Change of the behavior of alarm sounds
and ringtones preview, so that it matches
the design.

Additionally fixed lack of ringtones list
looping.

**Your checklist for this pull request**

Make sure that this PR:
- [x] Complies with our guidelines for contributions
- [ ] Has unit tests if possible
- [ ] Has documentation updated
- [x] Has changelog entry added

### CODE REVIEW

File: pure_changelog.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 * Changed responses in contacts and messages endpoints used to communicate with Center
 * Added SMS template call rejection window for case when no templates are defined
 * Changed unknown character glyph to the one matching the design
+* Changed alarm sounds and ringtones preview behavior to match the design
```
Comment (SP2FET) [inline pure_changelog.md:20]: it doesn't give any value to the end-user :P maybe it's possible to describe briefly how it should work?

**Review by SP2FET — APPROVED** [review]

File: pure_changelog.md (hunk 1 above)
Comment (Lefucjusz) [reply pure_changelog.md:20]: What would you say about something like `Changed alarm sounds and ringtones preview behavior - navigating through the list automatically switches playback to the currently selected one`? Isn't it too long as for the changelog entry?

Comment (SP2FET) [reply pure_changelog.md:20]: I'd say
`Fixed navigating through the ringtone preview list automatically switches playback to the currently selected one` or something like that

Comment (Lefucjusz) [reply pure_changelog.md]: Thanks, fixed!

**Review by kkleczkowski — APPROVED** [review]

**Pull request merged**